test/px-data-table-fixture.html

```diff
@@ -187,8 +185,32 @@
           table-rows="true"
           >
         </px-data-table>
-      </div>-->
+      </div>
 
+      <p>Smart Row Selection, client</p>
+      <div id="flexContainer2" style=" height:800px; border: 1px solid cyan; flex:1; display:flex; flex-direction:column; ">
+        <px-data-table
+          enable-column-resize="true"
+          greedy-height-with-scroll="true"
+          id="clientSmartRowSelectionWithLargeHeight"
```

Comment:
Apologies for not making this comment earlier:
Would you be open to changing this ID to `greedyHeightWithScrollLarge`, and the next one to `greedyHeightWithScrollDataRemote`?